src/build.py

```diff
@@ -1233,15 +1241,15 @@ def Summary(repos):
       buildbot.Link('lkgr.json', cloud.Upload(info_file, lkgr_file))
 
 
-def AllBuilds(use_asm=False):
+def AllBuilds(use_asm=False, run_tests=False):
```

Comment:
I think +1 for an options dict instead of function args here. I think it's fine for it to match the command line options, which is probably easier than arbitrarily inventing new option names.